wallet/src/ui/app/components/nft-display/index.tsx

```diff
@@ -29,14 +29,17 @@ function NFTDisplayCard({
     const { filePath, nftObjectID, nftFields, fileExtentionType } =
         useNFTBasicData(nftobj);
 
+    const name = nftFields?.name || nftFields?.metadata?.fields?.name;
```

Comment:
Is there any special reason using an independent metadata field to store NFT name?

Reply:
The reasoning behind it is that we are trying to be as generic as possible, but I guess some basic fields do make sense to have. In general, which fields do you see being part of the NFT object rather than the metadata object?

Reply:
name, description, image, [optional]animation_url, [optional] attributes

you can take metaplex for an example here [doc](https://docs.metaplex.com/programs/token-metadata/token-standard)

Reply:
Having a number of "special" (field name, type) pairs is the approach we've taken so far--cc @666lcz for more details on what we currently support.

Reply:
Hi @sblackshear @666lcz @nmboavida, we(Suiet Wallet) also noticed ecosystem projects have totally different practices on the same thing inside the metadata of the objects. So we create a draft standard/best practice for the ecosystem projects hoping to improve the forward compatibility and extendability.

Feel free to comment on our repo, we really hoping to push the ecosystem forward together.

https://std.suiet.app/
https://github.com/suiet/standard